docs/concepts/json.md

```diff
@@ -60,6 +60,62 @@ with one noticeable enhancement being that `jiter` supports deserialization of `
 In the future, `jiter` is intended to enable support validation errors to include the location
 in the original JSON input which contained the invalid value.
 
+### Partial JSON Parsing
+
+Starting in v2.7.0 and above, Pydantic's [JSON parser](https://docs.rs/jiter/latest/jiter/) offers support for partial JSON parsing, which is exposed via [`pydantic_core.from_json`][pydantic_core.from_json]. Here's an example of this feature in action:
+
+```py
+from pydantic_core import from_json
+
+partial_json_data = '["aa", "bb", "c'  # (1)!
+
+try:
+    result = from_json(partial_json_data, allow_partial=False)
+except ValueError as ve:
+    print(ve)  # (2)!
+    #> EOF while parsing a string at line 1 column 15
+
+result = from_json(partial_json_data, allow_partial=True)
+print(result)  # (3)!
+#> ['aa', 'bb']
+```
+
+1. The JSON list is incomplete - it's missing a closing `"]`
+2. When `allow_partial` is set to `False`, a parsing error occurs.
+3. When `allow_partial` is set to `True`, part of the input is deserialized successfully.
+
+This also works for deserializing partial dictionaries. For example:
+
+```py
+from pydantic_core import from_json
+
+partial_dog_json = '{"breed": "lab", "name": "fluffy", "friends": ["buddy", "spot", "rufus"], "age}'
+dog_dict = from_json(partial_dog_json, allow_partial=True)
+print(dog_dict)
+#> {'breed': 'lab', 'name': 'fluffy', 'friends': ['buddy', 'spot', 'rufus']}
+```
+
+!!! tip "Validating LLM Output"
+    This feature is particularly beneficial for validating LLM outputs.
+    We've written some blog posts about this topic, which you can find [here](#TODO: insert link filtered on LLMs).
+
+In future versions of Pydantic, we expect to expand support for this feature through either Pydantic's other JSON validation functions
+([`pydantic.main.BaseModel.model_validate_json`][pydantic.main.BaseModel.model_validate_json] and
+[`pydantic.type_adapter.TypeAdapter.validate_json`][pydantic.type_adapter.TypeAdapter.validate_json]) or model configuration. Stay tuned 🚀!
+
+### Caching Strings
+
+Starting in v2.7.0 and above, Pydantic's [JSON parser](https://docs.rs/jiter/latest/jiter/) offers support for string caching.
+The `cache_strings` setting is exposed via both [model config][pydantic.config.ConfigDict] and [`pydantic_core.from_json`][pydantic_core.from_json].
+
+The `cache_strings` setting can take any of the following values:
+
+* `True` or `'all'` (the default): cache all strings
+* `keys`: cache only dictionary keys
+* `False` or `none`: no caching
+
+Using the string caching feature results in performance improvements, but increases memory usage slightly.
```

Comment:
we should mention that:

1. Strings are cached using a fully associative cache with size [16,384](https://github.com/pydantic/jiter/blob/5bbdcfd22882b7b286416b22f74abd549c7b2fd7/src/py_string_cache.rs#L113), with a link to that code in rust
2. We should say that only strings where `len(string) < 64` (or is it `<=`?) are cached
3. Explain that there is some overhead to looking up the cache, which is normally worth it to avoid constructing strings, but if you know there will be very few repeated strings in your data, you might get a performance boost by disabling it.

Reply:
Done, thanks!